src/languages/es.ts

```diff
@@ -1981,7 +1982,31 @@ const translations = {
             es: {
                 label: 'Español',
             },
+            // eslint-disable-next-line @typescript-eslint/naming-convention
+            'pt-BR': {
+                label: 'Português (BR)', // @dont-translate
```

Comment:
Oh, sorry if I didn't update your section of the doc - we're not doing `@dont-translate` annotations anymore. Instead, we'll just provide a hard-coded list of things to not translate. So you can remove these annotations.